scripts/mergedEquippableUserJourney.ts

```diff
@@ -227,196 +227,166 @@ async function mintTokens(
-      },
+      0,
+      baseAddress,
+      `ipfs://gems/typeA/full.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into left slot for first type of gem
-      {
-        id: 2,
-        equippableGroupId: equippableRefIdLeftGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeA/left.svg`,
-      },
+      equippableRefIdLeftGem,
+      baseAddress,
+      `ipfs://gems/typeA/left.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into mid slot for first type of gem
-      {
-        id: 3,
-        equippableGroupId: equippableRefIdMidGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeA/mid.svg`,
-      },
+      equippableRefIdMidGem,
+      baseAddress,
+      `ipfs://gems/typeA/mid.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into left slot for first type of gem
-      {
-        id: 4,
-        equippableGroupId: equippableRefIdRightGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeA/right.svg`,
-      },
+      equippableRefIdRightGem,
+      baseAddress,
+      `ipfs://gems/typeA/right.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Full version for second type of gem, no need of refId or base
-      {
-        id: 5,
-        equippableGroupId: 0,
-        baseAddress: ethers.constants.AddressZero,
-        metadataURI: `ipfs://gems/typeB/full.svg`,
-      },
+      0,
+      ethers.constants.AddressZero,
+      `ipfs://gems/typeB/full.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into left slot for second type of gem
-      {
-        id: 6,
-        equippableGroupId: equippableRefIdLeftGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/left.svg`,
-      },
+      equippableRefIdLeftGem,
+      baseAddress,
+      `ipfs://gems/typeB/left.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into mid slot for second type of gem
-      {
-        id: 7,
-        equippableGroupId: equippableRefIdMidGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/mid.svg`,
-      },
+      equippableRefIdMidGem,
+      baseAddress,
+      `ipfs://gems/typeB/mid.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
+    await gem.addAssetEntry(
       // Equipped into right slot for second type of gem
-      {
-        id: 8,
-        equippableGroupId: equippableRefIdRightGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/right.svg`,
-      },
+      equippableRefIdRightGem,
+      baseAddress,
+      `ipfs://gems/typeB/right.svg`,
```

Comment:
Would you mind readding the keys?

*At this point I feel like Gollum and the keys are my ring*
![image](https://user-images.githubusercontent.com/8097439/202561813-472ab03c-33d1-4491-8034-7b402e176c4c.png)